lib/local-cluster/lib/Cardano/Wallet/Launch/Cluster/Cluster.hs

```diff
@@ -207,46 +212,47 @@ withCluster config@Config{..} faucetFunds onClusterStart = runClusterM config
             let pool0port :| poolPorts = NE.fromList (rotate poolsTcpPorts)
             let pool0 :| otherPools = configuredPools
 
-            runningPool0 <-
-                ContT
-                    $ operatePool pool0
-                    $ NodeParams
-                        genesisFiles
-                        cfgLastHardFork
-                        pool0port
-                        cfgNodeLogging
-                        cfgNodeOutputFile
+            passing ()
+                $ operatePool pool0
```

Comment:
```suggestion
            void
                $ ContT
                $ operatePool pool0
```

Reply:
?

Reply:
```haskell
                   void $ ContT
                        $ launchPools
                            others
                            genesisFiles
                            poolPorts
```
this is not compiling :-) . That's why we need to promote the `m a -> m a` to `(b -> m a) -> m a` 
```
lib/Cardano/Wallet/Launch/Cluster/Cluster.hs:248:27: error: [GHC-83865]
    * Couldn't match type: ClusterM a
                     with: a0 -> ClusterM a
      Expected: (a0 -> ClusterM a) -> ClusterM a
        Actual: ClusterM a -> ClusterM a
    * In the second argument of `($)', namely
        `launchPools others genesisFiles poolPorts'
      In the second argument of `($)', namely
        `ContT $ launchPools others genesisFiles poolPorts'
      In a stmt of a 'do' block:
        void $ ContT $ launchPools others genesisFiles poolPorts
    * Relevant bindings include
        onClusterStart :: RunningNode -> IO a
          (bound at lib/Cardano/Wallet/Launch/Cluster/Cluster.hs:182:43)
        withCluster :: Config
                       -> FaucetFunds -> (RunningNode -> IO a) -> IO a
          (bound at lib/Cardano/Wallet/Launch/Cluster/Cluster.hs:182:1)
```

Reply:
Ah, I see, sorry. 🙈 

How about this, then:

```hs
contT_ :: (m r → m r) → ContT m r
contT_ f = ContT $ \k → f (k ())
```

This would be in line with existing conventions (e.g. `for` vs `for_`) and the name `contT` establishes that this function is a variant of `ContT`. 🤔